packages/swap/src/routes/swap.ts

```diff
@@ -164,6 +179,7 @@ router.get(
       broadcastSucceededAt: swap?.egress?.broadcast?.succeededAt?.valueOf(),
       broadcastSucceededBlockIndex:
         swap?.egress?.broadcast?.succeededBlockIndex,
+      depositChannelExpiryTime: depositChannelExpiryTime?.valueOf(),
```

Comment:
i think we should return the block as well. That allows us to show the expiry block and give the user an timed estimate for full transparency

Reply:
It's not in the design for now, but guys are updating the design. If needed, I will add it too